src/components/products-filter-list/ClearAllButton.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
+
+const ClearAllButton = ({ forScreenSize }) => (
+  <button
+    className={`bg-unset border-none fw-normal link ${
+      forScreenSize === "md" ? "mx-1" : ""
+    }`}
+  >
+    Clear All
+  </button>
+);
+
+ClearAllButton.propTypes = {
+  forScreenSize: PropTypes.string,
+};
+
+ClearAllButton.defaultProps = {
+  forScreenSize: "",
+};
```

Comment:
Appreciation - good thing that you have used data type & default value for props